src/pages/SignIn.js

```diff
@@ -89,6 +89,7 @@ const SignIn = ({ network, initNetwork, eraseNetwork }) => {
         className='form-input'
         type='password'
         required
+        autoFocus
```

Comment:
Amazingly, I always thought that this was supposed to be `autofocus` in React, but apparently it's camel-cased??